src/nodes/image.ts

```diff
@@ -117,8 +117,9 @@ export class ImageNode extends GraphicsNode {
       format = mimeTypeParts[1]
 
       data = match[5]
+
       if (match[4] === 'base64') {
-        data = data.replace(/\s/g, '')
+        data = data.replace(/\s/g || /\n/g, '')
```

Comment:
This doesn't work. If you want to replace \s or \n, you need to put them in the same regex. But \n is part of \s, so this change is not necessary.